### PR TITLE
4.0 updates

### PR DESCRIPTION
**Pull request by jyemin**
Patch build: https://evergreen.mongodb.com/version/5b27da53e3c3312e46c49660

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -44,7 +44,7 @@ configure(subprojects.findAll { it.name != 'util' }) {
     evaluationDependsOn(':util')
 
     group = 'org.mongodb'
-    version = '1.9.0-beta1'
+    version = '1.9.0-SNAPSHOT'
```
Comment (jyemin) [inline build.gradle]: Back to snapshot until the next release

File: driver/build.gradle [hunk 2]
```diff
@@ -20,7 +20,7 @@ description = "A Reactive Streams implementation of the MongoDB Java driver"
 archivesBaseName = 'mongodb-driver-reactivestreams'
 
 dependencies {
-    compile 'org.mongodb:mongodb-driver-async:3.8.0-beta2'
+    compile 'org.mongodb:mongodb-driver-async:3.8.0-SNAPSHOT'
```
Comment (jyemin) [inline driver/build.gradle]: Until we do another release of the driver

File: driver/src/main/com/mongodb/reactivestreams/client/MongoClient.java [hunk 3]
```diff
@@ -118,6 +120,102 @@
      */
     <TResult> ListDatabasesPublisher<TResult> listDatabases(ClientSession clientSession, Class<TResult> clazz);
 
+        /**
```
Comment (jyemin) [inline driver/src/main/com/mongodb/reactivestreams/client/MongoClient.java]: Will fix

**Comment by jyemin** [comment]
New patch build: https://evergreen.mongodb.com/version/5b27e28de3c3312e46c6267f

**Comment by jyemin** [comment]
New patch build: https://evergreen.mongodb.com/version/5b28056be3c3312e46cead1d

File: driver/src/main/com/mongodb/reactivestreams/client/ChangeStreamPublisher.java [hunk 4]
```diff
@@ -49,6 +50,20 @@
      */
     ChangeStreamPublisher<TResult> resumeAfter(BsonDocument resumeToken);
 
+    /**
+     * The change stream will only provides changes that occurred after the specified timestamp.
```
Comment (rozza) [inline driver/src/main/com/mongodb/reactivestreams/client/ChangeStreamPublisher.java]: Minor change to: The change stream will only provide changes that occurred at or after the specified timestamp.

**Pull request closed without merging**